### ISSUE TITLE
Ability to add/remove someone as a triager

### ISSUE THREAD

**Issue by carols10cents** [eeyorebot/eeyore#4]
Given someone's github handle, there should be a way to specify a person as a triager who is allowed to ask eeyore to do things on their behalf.

One idea I had for this is having a read-only github team (which normally does NOT allow those people to modify issues) with a special name (ex: 'Triagers' or 'Triage' or something) that eeyore would recognize.

Or this could just be a list of handles that eeyore keeps track of, and it doesn't integrate with github teams at all.  ¯\_(ツ)_/¯

**Comment by steveklabnik** [comment]
yeah, I'm not sure where I fall on this one. I think I'd support the teams approach, mostly because that way, people get credit on their github profiles for being a member of an org and on a team.

What about letting eeyore create this team if it doesn't already exist?

**Comment by carols10cents** [comment]
> What about letting eeyore create this team if it doesn't already exist?

OOoooh that would be nice. It might also be nice if, instead of having to add people through eeyore's web interface, an admin on the repo could say in any issue "hey @eeyorebot, add so-and-so as a triager" and bam it's done. That's probably nice to have though.

**Comment by notriddle** [comment]
Admittedly, if you do it with a team, Eeyore will have to be a member of the org to see it.
